managers/tests/test_unit_managers_apiview.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+
+from rest_framework.reverse import reverse
+from rest_framework.test import APIRequestFactory
+
+from django.test import TestCase
+
+from users.models import CustomUser
+from managers import views
+from managers.models import Project
+
+
+class ProjectTest(TestCase):
+    def setUp(self):
+        self.user = CustomUser(
+            email="testuser@codepoets.it",
+            first_name='John',
+            last_name='Doe',
+            country='PL',
+            user_type=CustomUser.UserType.ADMIN.name,
+        )
+        self.user.password = 'newuserpasswd'
+        self.user.set_password(self.user.password)
```

Comment:
If you call `set_password()`, the value `self.user.password` is overwritten with a hashed password, so there is no need to set it directly in the line above.

Reply:
Done